plugins/module_utils/azure_rm_common.py

```diff
@@ -1203,6 +1203,7 @@ def containerregistry_client(self):
         if not self._containerregistry_client:
             self._containerregistry_client = self.get_mgmt_svc_client(ContainerRegistryManagementClient,
                                                                       base_url=self._cloud_environment.endpoints.resource_manager,
+                                                                      is_track2=True,
                                                                       api_version='2017-10-01')
```

Comment:
Now that you've all upgraded to Track2, you can upgrade this version to a newer version. Thank you very much!
```suggestion
                                                                      api_version='2019-09-01')
```